.github/workflows/ci.yml

```diff
@@ -15,10 +15,9 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-20.04
-        python-version:
-          - 3.8
-        toxenv: [ py38 ]
+        - ubuntu-20.04
+        python-version: ['3.8', '3.11']
+        toxenv: [py38]
```

Comment:
toxenv needs to be updated so tests run on `Python 3.11` as well. Right now, tests are running only on `Python 3.8` in both shards.